db/data/students.csv

```diff
@@ -1,114 +1,114 @@
-c8mahler,Mahler,Gustav
-c9magnar,Magnard,Alberic
+c8mahler,Mahler,Gustav,section_b
```

Comment:
The section names should be user facing. You can use the Toronto convention, `LEC0101` and `LEC0201`.